src/components/Envelope.vue

```diff
@@ -59,26 +59,47 @@
 		</template>
 		<template #actions>
 			<EnvelopePrimaryActions v-if="!moreActionsOpen">
-				<ActionButton icon="icon-starred"
+				<ActionButton
 					class="action--primary"
 					:close-after-click="true"
 					@click.prevent="onToggleFlagged">
+					<template #icon>
+						<Star v-if="iconFavorite"
+							:title="t('mail', 'Favorite')"
+							:size="20" />
+						<StarOutline v-else
+							:title="t('mail', 'Unfavorite')"
+							:size="20" />
```

Comment:
Could we swap both icons with each other? Then it would be more consistent with the read/unread icons.